src/server/routes/webhooks/commands.rs

```diff
@@ -15,7 +15,17 @@ pub fn ping(data: &Data, issue: &Issue) -> Result<()> {
 }
 
 pub fn run(host: &str, data: &Data, issue: &Issue, args: RunArgs) -> Result<()> {
-    let name = get_name(&data.db, issue, args.name)?;
+    let name = {
+        let name_supplied = args.name.is_some();
+        let name = get_name(&data.db, issue, args.name)?;
+        if name_supplied {
+            name
+        } else {
+            let new_name = auto_increment_experiment_name(&data.db, &name)?;
```

Comment:
By passing as argument the previously stored name this is generating weird results, like `last-name-1-1-1` (like `pr-123-1-1-1-1`).